client/api.go

```diff
@@ -16,26 +14,38 @@ const (
 )
 
 func Host() string {
+	h, err := HostURL()
+	if err != nil {
+		panic(err)
```

Comment:
would prefer to output this on `os.Stderr` and then `os.Exit(1)` (preferably from bubbling the error up) -- users are specifying the `API_URL` directly and we should expect it to be wrong, `panic` makes more sense in cases where something shouldn't happen.